environment.yml

```diff
@@ -6,6 +6,7 @@ dependencies:
   - codecov
   - coverage
   - gemmi
+  - geomstats
```

Comment:
Tests are failing on this. Maybe you need to add some channels as in https://github.com/geoffwoollard/compSPI/blob/stash_simulate/environment.yml#L2

Reply:
The issue was that geomstats can't come from conda (it's pip only). Fixed.

Reply:
For ref, it can come from conda as in the link above https://github.com/geoffwoollard/compSPI/blob/stash_simulate/environment.yml#L17

see how the conda channels are different.

anyways it's working, so fine I think. but why not do it just like the link since that comes from Nina's past project, I believe.